grizzly/common/storage.py

```diff
@@ -445,25 +444,24 @@ def load_single(cls, path, adjacent=False, load_assets=True, copy=True):
         # load all adjacent data from directory
         if adjacent:
             asset_path = info.get("assets_path", None)
-            for dpath, _, files in walk(entry_point.parent):
-                dpath = Path(dpath)
+            for entry in Path(entry_point.parent).rglob("*"):
+                if not entry.is_file():
+                    continue
+                location = entry.relative_to(entry_point.parent).as_posix()
                 # ignore asset path
-                if asset_path and dpath.samefile(entry_point.parent / asset_path):
+                if asset_path and location.startswith(asset_path):
```

Comment:
Minor nit - you could also use samefile here.

Reply:
@tysmith shared this over matrix - [Shutil only supports path objects in 3.9](https://docs.python.org/3/library/shutil.html#shutil.move)

Reply:
This is checking if a file is in `asset_path` not if it is `asset_path`